display-scene/src/main/java/com/esri/arcgisruntime/sample/displayscene/MainActivity.java

```diff
@@ -0,0 +1,71 @@
+/* Copyright 2017 Esri
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.esri.arcgisruntime.sample.displayscene;
+
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+
+import com.esri.arcgisruntime.mapping.ArcGISScene;
+import com.esri.arcgisruntime.mapping.ArcGISTiledElevationSource;
+import com.esri.arcgisruntime.mapping.Basemap;
+import com.esri.arcgisruntime.mapping.Surface;
+import com.esri.arcgisruntime.mapping.view.Camera;
+import com.esri.arcgisruntime.mapping.view.SceneView;
+
+import static com.esri.arcgisruntime.sample.displayscene.R.id.sceneView;
+
+public class MainActivity extends AppCompatActivity {
+
+    private SceneView mSceneView;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+        // inflate MapView from layout
+        mSceneView = (SceneView) findViewById(sceneView);
+        // create a scene and add a basemap to it
+        ArcGISScene agsScene = new ArcGISScene();
+        agsScene.setBasemap(Basemap.createImagery());
+        mSceneView.setScene(agsScene);
+
+        // add base surface for elevation data
+        ArcGISTiledElevationSource elevationSource = new ArcGISTiledElevationSource(
+                getResources().getString(R.string.elevation_image_service));
+        Surface surface = new Surface();
+        surface.getElevationSources().add(elevationSource);
+        agsScene.setBaseSurface(surface);
```

Comment:
A new ArcGISScene already has a Surface, so you can just add the new elevation source to the existing surface.